packages/hash/api/src/model/knowledge/org.model.ts

```diff
@@ -0,0 +1,268 @@
+import { GraphApi } from "../../graph";
+import {
+  OrgModel,
+  EntityModel,
+  EntityModelCreateParams,
+  AccountFields,
+  EntityTypeModel,
+} from "..";
+import {
+  generateSchemaBaseUri,
+  generateWorkspaceEntityTypeSchema,
+  generateWorkspacePropertyTypeSchema,
+  workspaceAccountId,
+  workspaceTypesNamespaceUri,
+} from "../util";
+
+// Generate the schema for the organization name property type
+export const orgNamePropertyType = generateWorkspacePropertyTypeSchema({
+  title: "Organization Name",
```

Comment:
I know it doesn't matter a lot right now, but why are we calling this "Organization Name" rather than something like "name"?

Reply:
Good question. Wouldn't we want to have a property type that's general enough to be used for either Users or Orgs then? (User defines "Preferred name")

Reply:
Hm, let's not change right now. Perhaps we'll need to discover the answers to this. We probably want different property types as I think they'll have different constraints on them, but we'll want to be able to identify them as being semantically the same thing (i.e. a name), so we can wait until we explore mappings further.